inference-engine/tests/functional/plugin/cpu/single_layer_tests/convolution_backprop_data.cpp

```diff
@@ -115,6 +115,13 @@ const SizeVector numOutChannels_Planar = { 6 };
 /* ============= Deconvolution params (blocked layout) ============= */
 const SizeVector numOutChannels_Blocked = { 64 };
 
+/* ======== Deconvolution params (runtimeError) ======== */
```

Comment:
I don't see big point in updating cpu specific instances since there is no any dependency on fusing or layouts. My recommendation is to update this one:  https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/tests/functional/plugin/cpu/shared_tests_instances/single_layer_tests/convolution_backprop_data.cpp

Reply:
Fixed in 2ece3c7ae13a53531ad2b29e7c2b8b7790e4c81d. Thank you!